build.sc

```diff
@@ -105,12 +105,6 @@ class chisel3CrossModule(val crossScalaVersion: String) extends CommonModule wit
   }
 
   object test extends Tests with TestModule.ScalaTest {
-    override def scalacPluginClasspath = m.scalacPluginClasspath
-
-    override  def scalacOptions = T {
-      super.scalacOptions() ++ Agg("-P:chiselplugin:genBundleElements")
-    }
-
```

Comment:
You don't want to delete the `scalacOption` because we **only** specify this argument for the tests, not for main Chisel (because it would be a binary incompatible change on a minor release, we'll make it mandatory in 3.6)

Reply:
yep, I hit this issue, I need more time to debug the mill build system.